src/modules/twap/state/twapPartOrdersAtom.ts

```diff
@@ -40,7 +45,10 @@ export const updatePartOrdersAtom = atom(null, (get, set, nextState: TwapPartOrd
     acc[parentId] = items.map((item) => {
       return {
         ...item,
-        isCreatedInOrderBook: currentItemsMap[item.uid]?.isCreatedInOrderBook || item.isCreatedInOrderBook,
+        ...virtualFields.reduce<Partial<TwapPartOrderItem>>((acc, val) => {
```

Comment:
Would be great to add some comments explaining what is happening here too.

Reply:
Added more comments